mozilla/bedrock
# update nav to remove existing nav styles

## One-line summary

This PR refactors the new navigation to remove old styles from the current nav. This is one of the steps to reduce new base CSS bundle size.

## Significant changes and points to review



## Issue / Bugzilla link

https://github.com/mozilla/bedrock/issues/15120
https://github.com/mozilla/bedrock/issues/15196

## Testing
1. Set `M24_NAVIGATION_AND_FOOTER` switch to be `on`
2. Check out this PR
3. Review navigation styles, it should look and function exactly like before refactoring
